tox.ini

```diff
@@ -10,6 +10,7 @@ description = Runs unittests
 deps=
     -rtest-requirements.txt
 commands=
+    {envpython} -m pip install types-paramiko
```

Comment:
This can go to ``deps``

Reply:
Sure got it, it can be simply added to the `test-requirements.txt`.